### PR TITLE
Update Readme.md

### PR DESCRIPTION
**Pull request by infinyte7**
## Purpose / Description
Updating Readme.md with more info about AnkiDroid.

## Fixes
Fixes _Link to the issues._

## Approach
Badge generated using 
http://img.shields.io/

## How Has This Been Tested?
Tested on Chrome browser

## Learning (optional, can help others)
http://img.shields.io/

## Checklist
_Please, go through these checks before submitting the PR._

- [ x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [ x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [ x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ x ] You have commented your code, particularly in hard-to-understand areas
- [ x ] You have performed a self-review of your own code

### CODE REVIEW

**Comment by infinyte7** [comment]
More ideas would be appreciated.

**Comment by david-allison-1** [comment]
Live preview: https://github.com/infinyte7/Anki-Android/tree/update-readme

File: README.md [hunk 1]
```diff
@@ -8,7 +21,11 @@ Install directly from [here on github](https://github.com/ankidroid/Anki-Android
 automatically by installing via an app store:
 
 <a href="https://play.google.com/store/apps/details?id=com.ichi2.anki&utm_source=global_co&utm_medium=prtnr&utm_content=Mar2515&utm_campaign=PartBadge&pcampaignid=MKT-Other-global-all-co-prtnr-py-PartBadge-Mar2515-1"><img alt="Get it on Google Play" src="https://play.google.com/intl/en_us/badges/images/generic/en-play-badge.png" height="50px"/></a>
-<a href="https://f-droid.org/repository/browse/?fdid=com.ichi2.anki"><img alt="F-Droid" src="https://upload.wikimedia.org/wikipedia/commons/0/0d/Get_it_on_F-Droid.svg" height="50px"/></a>
+<a href="https://f-droid.org/repository/browse/?fdid=com.ichi2.anki"><img alt="F-Droid" src="https://upload.wikimedia.org/wikipedia/commons/4/40/%22Get_it_on_F-Droid%22_Badge.svg" height="50px"/></a>
+
```
Comment (mikehardy) [inline README.md:25]: Now that I'm auto-publishing releases to Amazon, it's more reliable to say we are available there - https://developer.amazon.com/support/legal/tuabg

Appropriate image: https://images-na.ssl-images-amazon.com/images/G/01/mobile-apps/devportal2/res/images/amazon-appstore-badge-english-black.png

Link: http://www.amazon.com/gp/product/B004L1G7T2/ref=mas_pm_ankidroid_flashcards

File: README.md [hunk 2]
```diff
@@ -1,5 +1,18 @@
+<p align="center"><img src="https://raw.githubusercontent.com/ankidroid/Anki-Android/master/AnkiDroid/src/main/res/drawable-hdpi/anki.png"></img></p>
+<h1 align="center">AnkiDroid</h1>
+
 A semi-official port of the open source [Anki](http://ankisrs.net/index.html) spaced repetition flashcard system to Android. Memorize anything with AnkiDroid!
 
+<p align="center">
+<a href="https://github.com/ankidroid/Anki-Android/releases"><img src="https://img.shields.io/github/v/release/ankidroid/Anki-Android" alt="release"/></a>
+<a href="https://travis-ci.org/github/ankidroid/Anki-Android"><img src="https://img.shields.io/travis/ankidroid/Anki-Android" alt="build"/></a>
+<a href="https://github.com/ankidroid/Anki-Android/issues"><img src="https://img.shields.io/github/issues/ankidroid/Anki-Android" alt="issues"/></a>
```
Comment (mikehardy) [inline README.md:9]: I think flow rate is more important than open issues, I'd switch it for commits/month https://shields.io/category/activity

File: README.md (hunk 1 above)
Comment (infinyte7) [reply README.md:26]: Image to be adjusted with size of other store badge. I will do it.

**Comment by mikehardy** [comment]
Now I'm just being nitpicky :-) but when is it possible to have the badges for the app stores all have the same dimensions? Just expanding the other two to match the amazon one seems best, they look a little cramped right now anyway. This looks cool though! I'm really liking it now

**Comment by infinyte7** [comment]
Yes, Same height and width, I will update it.

**Comment by infinyte7** [comment]
https://img.shields.io/discord/368267295601983490
Widget is disabled. Mod may enable that server widget settings. 
https://vimeo.com/364220040

**Comment by david-allison-1** [comment]
@infinyte7 https://discord.gg/qjzcRTx should be good.

I love this change by the way, thank you!!

**Comment by infinyte7** [comment]
That image for adding join discord sever ?
Because clicking on chat badge will lead to invite link.

**Comment by infinyte7** [comment]
Is it good to add reddit and discord under "Chat" in Readme.md like install?

**Comment by david-allison-1** [comment]
Yeah, Reddit is a good shout!

**Comment by infinyte7** [comment]
To be updated.

**Comment by infinyte7** [comment]
Updated the readme. Also created screenshots like this. But it needs more improvements.



More at:
[](https://github.com/infinyte7/AnkiDroid-Screenshots)

File: README.md [hunk 3]
```diff
@@ -1,7 +1,6 @@
-<p align="center"><img src="https://raw.githubusercontent.com/ankidroid/Anki-Android/master/AnkiDroid/src/main/res/drawable-hdpi/anki.png"></img></p>
-<h1 align="center">AnkiDroid</h1>
-
-A semi-official port of the open source [Anki](http://ankisrs.net/index.html) spaced repetition flashcard system to Android. Memorize anything with AnkiDroid!
+<p align="center">
+<img src="https://github.com/infinyte7/AnkiDroid-Screenshots/blob/master/banner_readme.png?raw=true"/>
```
Comment (david-allison-1) [inline README.md:2]: Is it possible to perform this as a relative URL? It would be ideal to only reference local assets in the readme

**Comment by david-allison-1** [comment]
Wow!

Maybe move the chat links up to the top... but wow

File: README.md (hunk 3 above)
Comment (infinyte7) [reply README.md:2]: It is possible.

**Comment by mikehardy** [comment]
OMG that is so much better than the previous "lacks common sense" one but I was too shy to ask :-) :-)

I want to merge this right away but will wait until you guys agree you're happy with it.

I will find a way to put this in the play store images as well.

**Comment by david-allison-1** [comment]
I'm more than happy to see this merged.

**Comment by mikehardy** [comment]
Any time you like it @infinyte7 just let me know then, I will merge it when you are happy. :muscle:

**Comment by infinyte7** [comment]
Yes, I am just uploading the image to AnkiDroid repo.

**Comment by infinyte7** [comment]
Read to merge. Working on mobile also.

File: README.md [hunk 4]
```diff
@@ -1,14 +1,59 @@
...
+</a>
+
+<a href="https://f-droid.org/repository/browse/?fdid=com.ichi2.anki">
+    <img alt="Get it on F-Droid" height="64"
+        src="https://upload.wikimedia.org/wikipedia/commons/4/40/%22Get_it_on_F-Droid%22_Badge.svg">
+</a>
+
+<a href="http://www.amazon.com/gp/product/B004L1G7T2/ref=mas_pm_ankidroid_flashcards">
+    <img alt="Get it on Amazon app store" height="64"
+        src="https://raw.githubusercontent.com/infinyte7/app-store-badge/master/Amazon-app-store.png"/>
```
Comment (mikehardy) [inline README.md:33]:
```suggestion
        src="https://images-na.ssl-images-amazon.com/images/G/01/mobile-apps/devportal2/res/images/amazon-appstore-badge-english-black.png"/>
```

I think should be this URL?

File: README.md [hunk 5]
```diff
@@ -19,3 +64,15 @@ Contribute
 ----------
 You can contribute to AnkiDroid by beta testing, translating, or submitting code. 
 See the [contribution wiki page](https://github.com/ankidroid/Anki-Android/wiki/Contributing) for more info.
+
+Join Us On
+----------
+
+<a href="https://discord.gg/qjzcRTx"><img src="https://raw.githubusercontent.com/infinyte7/icons-and-badges/master/discord_logo_color.png" height="50px"/></a>
```
Comment (mikehardy) [inline README.md:71]: these should be external references to their official logos, or should be internal to ankidroid/Anki-Android repo - I have a preference for external if possible, unless / until they prove flaky

Comment (infinyte7) [reply README.md:71]: It would be better to add that logo and image to ankidroid docs. Because it increase repo size, and then apk size.

Comment (infinyte7) [reply README.md:71]: Should I create PR in ankidroid docs for images and logo?

File: README.md (hunk 4 above)
Comment (infinyte7) [reply README.md:33]: I have edited the image to match the space lining with other store badge. Play store and F-Droid have same space around their body. But Amazon app store have more extra spaces, which make it align little below the other store badges.

File: README.md (hunk 5 above)
Comment (mikehardy) [reply README.md:71]: Your assumption is flawed, the APK build process is quite specific, and there is no reason these will go in the APK unless they are in AnkiDroid/src/main/res/... - if you put them in docs (https://github.com/ankidroid/Anki-Android/tree/master/docs/graphics), they are just there, not in the APK - could make a "logos" folder there or something

File: README.md (hunk 4 above)
Comment (mikehardy) [reply README.md:33]: Ah, I see what you mean for the amazon logo then - put it in the docs tree as mentioned separately as well then, and it'll be edited as you like but also in the repo with the readme

Comment (infinyte7) [reply README.md:33]: Also tried css style to align it properly but Readme.md file get sanitized. And css removed.
https://stackoverflow.com/questions/51956361/custom-css-file-for-readme-md-in-a-github-repo

File: README.md (hunk 5 above)
Comment (infinyte7) [reply README.md]: Learned it. Thank you

**Comment by infinyte7** [comment]
;-), Now, I am creating screenshots for play store.

I will let you know when it get finished.

Also, need ideas about adding header to images.

I have created these headers,
- Create Note
- Edit Note
- Edit Template
- Type Answer
- Night Mode
- Statistics
- Card browser
- Whiteboard
- Custom Layout
- Change background

Idea 1:
One line header to all images

Idea 2:
Header and below subtitle explaining the header. 

I prefer one line header.

**Comment by mikehardy** [comment]
I like one line as well, the less text the better both from an international perspective (the words will be meaningless for people that don't speak English) and because text is small in many cases

But, in general do your best, trust your intuition. I will try not to micromanage - I don't really have time for it anyway ;-)

**Comment by infinyte7** [comment]
Okay, Thank you

**Comment by mikehardy** [comment]
launched :rocket:

File: docs/graphics/logos/ankidroid_logo.png
Comment (Arthur-Milchior) [inline docs/graphics/logos/ankidroid_logo.png:1]: By any luck, do you remember when the original image come from. I assume you have not created the logo yourself. @krmanik

**Pull request merged**